### ISSUE TITLE
Keyboard controls creates invalid numbers

### ISSUE THREAD

**Issue by tejovanthn** [siege-media/contrast-ratio#15]
Holding on UP-ARROW/DOWN-ARROW doesn't update the hexadecimal properly, but appends (negative/invalid) numbers.

**Comment by LeaVerou** [comment]
Yeah, that's a well known bug with [Incrementable](https://github.com/leaverou/Incrementable), which this project uses…

**Comment by peisenmann** [comment]
The site is a great, and this doesn't ruin it for me, but I did just test it out and see that it's still broken for hex.

**Comment by ollie-iterators** [comment]
This is also an issue with opacity
